tests/components/linear_garage_door/test_config_flow.py

```diff
@@ -65,54 +65,58 @@ async def test_form(hass: HomeAssistant) -> None:
 async def test_reauth(hass: HomeAssistant) -> None:
     """Test reauthentication."""
 
-    entry = await async_init_integration(hass)
-
-    result = await hass.config_entries.flow.async_init(
-        DOMAIN,
-        context={
-            "source": config_entries.SOURCE_REAUTH,
-            "entry_id": entry.entry_id,
-            "title_placeholders": {"name": entry.title},
-            "unique_id": entry.unique_id,
-        },
-        data=entry.data,
-    )
-    assert result["type"] == FlowResultType.FORM
-    assert result["step_id"] == "user"
-
     with patch(
-        "homeassistant.components.linear_garage_door.config_flow.Linear.login",
-        return_value=True,
-    ), patch(
-        "homeassistant.components.linear_garage_door.config_flow.Linear.get_sites",
-        return_value=[{"id": "test-site-id", "name": "test-site-name"}],
-    ), patch(
-        "homeassistant.components.linear_garage_door.config_flow.Linear.close",
-        return_value=None,
-    ), patch(
-        "uuid.uuid4",
-        return_value="test-uuid",
+        "homeassistant.components.linear_garage_door.coordinator.LinearUpdateCoordinator._async_update_data",
```

Comment:
We shouldn't patch internals, patch the library only

Reply:
Without this, the test was failing. See here: https://discord.com/channels/330944238910963714/330990195199442944/1206385236145864754

Reply:
I don't see that you are patching `get_devices` so perhaps do that and it would work (if now that's a call within the update)

Reply:
I patch it here on line 28: https://github.com/home-assistant/core/blob/dev/tests/components/linear_garage_door/util.py#L28

Is that not enough?

Reply:
Why don't you do like this instead to skip setup on this test as it's not really needed for this test?
I would have to dig a bit more to understand why you get some timeout really.
```python
patch(
        "homeassistant.components.linear_garage_door.async_setup_entry",
        return_value=True,
    )
```